source/adminguide/service_offerings.rst

```diff
@@ -69,14 +74,20 @@ available offerings when they create a new VM. Based on the user’s
 selected offering, CloudStack emits usage records that can be integrated
 with billing systems.
 
-Some characteristics of service offerings must be defined by the CloudStack
-administrator, and others can be left undefined so that the end-user can
-enter their own desired values. This is useful to reduce the number of
-offerings the CloudStack administrator has to define. Instead of defining a
-compute offering for every imaginable combination of values that a user
+Compute offerings may be "fixed", "custom constrained" or "custom unconstrained".
+
+In fixed offering the Number of CPUs, Memory and CPU frequecy in each service
+offerings are predefined by the CloudStack administrator, in custom unconstrained
+offerings they are left undefined so that the end-user can enter their own desired
+values when creating a guest instance. Since 4.13 custom constrained offerings have
+been introduced to allow the end-user to enter the number of CPUs and memory
+required within constraints set by the administrator.  The constraints can be 
```

Comment:
Double space here, `administrator.  The constraints`. Also in the line below.
Although not showing in the docs html

Reply:
a double space after a full stop is the 'proper' way to write.  but a lot of markup languages were written by Americans [TIC] so they ignore them.  - please send any complaints to:
The Queen
England.